dali/imgcodec/decoders/nvjpeg/nvjpeg2k_helper.h

```diff
@@ -15,6 +15,8 @@
 #ifndef DALI_IMGCODEC_DECODERS_NVJPEG_NVJPEG2K_HELPER_H_
 #define DALI_IMGCODEC_DECODERS_NVJPEG_NVJPEG2K_HELPER_H_
 
+#if NVJPEG2K_ENABLED
```

Comment:
I would rather surround the include call than having this inside the file. Not very important though

Reply:
Yes, that's the plan